### PR TITLE
Increase max metric limit

### PR DESCRIPTION
**Pull request by aahmed-2**
Increase meson option for total maximum Metrics/MetricProperties
limit from 200 to 300. Meson option name is max-reading-parameters.

Testing:
Monitored CPU usage of bmcweb for both rainier and everest systems.
Bmcweb spike in usage for around 2 seconds with max MetricProperty MRD.
MRD was periodic on 15 interval with 300 MetricProperties averaged over
30 seconds on everest system with 360 sensors. Max bmcweb usage was 9%
for brief instant when MRD post call is made.

Signed-off-by: Ali Ahmed <ama213000@gmail.com>

### CODE REVIEW

**Comment by aahmed-2** [comment]
Defect number for this is SW552143

File: meson_options.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 option('buildtest', type: 'boolean', value: true, description: 'Build tests')
 option('max-reports', type: 'integer', min: 1, value: 10,
        description: 'Max number of Reports')
-option('max-reading-parameters', type: 'integer', min: 1, value: 200,
+option('max-reading-parameters', type: 'integer', min: 1, value: 300,
```
Comment (gtmills) [inline meson_options.txt:4]: Can we pass this in? In a telemetry bb append ?

**Review by gtmills — APPROVED** [review]
This works for now. Long term let's move to a bbappend

**Comment by gtmills** [comment]
This is upstream at https://gerrit.openbmc.org/c/openbmc/openbmc/+/54788

**Pull request merged**